src/workloads/sharding/MongosMerging.yml

```diff
@@ -0,0 +1,151 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/product-perf"
+Description: |
+  This workload runs different types of aggregations where the query results
```

Comment:
What metrics should we be looking at?

Reply:
Updated the docs to clarify that we're looking for a failure here